index.html

```diff
@@ -138,23 +138,18 @@
         </ul>
     </nav>
 
-    <div id="contentBody">
-        <section class="content-right-now">
-            <h2>På Bio just nu</h2>
-            <ul id="just-nu">
-        
-                <!-- card template
-                <div>
-                    <img src="src/assets/avatar.jpeg" alt="">
-                     <h3>Lorem ipsum dolor sit amet consectetur </h3>
-                 </div>
-                -->
-        
-            </ul>
-        </section>
+    <section class="movie-content">
+        <div class="showing-now">
+        <h1 class="heading-1"></h1>
+        <div class="movies-now "></div>
     </div>
-    
-    
+    <div class="showing-later">
+        <h1 class="heading-2"></h1>
+        <div class="movies-later "></div>
+    </div>
+    </section>
```

Comment:
This section need to be inside a div in order to the other content pages to work. I see this PR is quite far behind current main.

My suggestion is to do a pull from main and then make new commits to this branch, so the PR is updated. Currently these codes might be missing from main since I removed them by accident in a previous PR. Added them back in my current PR. But just make sure they are in a div with class="content-body" and id="contentBody".

Reply:
Ok, let me check that out